projects/fsastorefrontlib/src/cms-components/banner/enriched-responsive-banner.component.html

```diff
@@ -1,12 +1,35 @@
 <ng-container *ngIf="component.data$ | async as data">
-  <div class="enriched-banner">
+  <div *ngIf="configStyles?.textBox" class="enriched-banner">
     <div class="image-wrapper">
       <cx-media [container]="data.media"></cx-media>
     </div>
-    <div class="enriched-banner-text">
-      <span class="enriched-banner-title">{{ data.headingText }}</span>
-      <span class="enriched-banner-details">
-        <a class="enriched-banner-styled-text" [routerLink]="data.urlLink">
+    <div
+      class="enriched-banner-text"
+      [ngStyle]="bannerStyles"
+      [ngClass]="bannerClasses"
+    >
+      <span
+        *ngIf="configStyles?.textBoxTitle"
```

Comment:
This could be changed to `data.headingText` so it won't be displayed if user omits it from impex.